lighthouse-core/audits/cache-start-url.js

```diff
@@ -39,20 +39,23 @@ class CacheStartUrl extends Audit {
    */
   static audit(artifacts) {
     let cacheHasStartUrl = false;
+    const manifest = artifacts.Manifest && artifacts.Manifest.value;
+    const cacheContents = artifacts.CacheContents;
+    const baseURL = artifacts.URL;
 
-    if (!(artifacts.Manifest &&
-          artifacts.Manifest.value &&
-          artifacts.Manifest.value.start_url &&
-          Array.isArray(artifacts.CacheContents) &&
-          artifacts.URL)) {
+    if (!(manifest && manifest.start_url && manifest.start_url.raw)) {
```

Comment:
can just be `if (!(manifest && manifest.start_url.raw)) {` (if there is a manifest, there is always a `manifest.start_url`)